subprojects/gradle/impact-shared/src/main/java/com/avito/impact/configuration/AndroidTestConfiguration.kt

```diff
@@ -9,14 +9,14 @@ class AndroidTestConfiguration(module: InternalModule) : SimpleConfiguration(mod
 
     override val isModified: Boolean by lazy {
         dependencies.any { it.isModified }
-            || module.testConfiguration.isModified
+            || module.implementationConfiguration.isModified
```

Comment:
`androidTest` doesn't depend on `test` sources.
So If `test` sources are changed then `androidTest` will not be affected